File: app/src/main/java/com/stormbreakerstudios/android/challenge/SplashActivity.java
package com.stormbreakerstudios.android.challenge;

import android.os.Bundle;

public class SplashActivity extends BaseActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setTransparentStatusBarForKitKatAndAbove();
        setContentView(R.layout.activity_splash);
        navigator.navigateToOnboardingScreen(this);
        finish();
    }
}

File: app/src/main/java/com/stormbreakerstudios/android/challenge/SplashActivity.kt
package com.stormbreakerstudios.android.challenge

import android.os.Bundle


class SplashActivity : BaseActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setTransparentStatusBarForKitKatAndAbove()
        setContentView(R.layout.activity_splash)
        navigator.navigateToOnboardingScreen(this)
        finish()
    }
}